synapse/rest/client/v2_alpha/keys.py

```diff
@@ -149,6 +151,52 @@ def on_POST(self, request):
         defer.returnValue((200, result))
 
 
+class KeyChangesServlet(RestServlet):
+    """Returns the list of changes of keys between two stream tokens (may return
+    spurious results).
```

Comment:
Maybe explain why the results might be spurious and the nature of their spuriosity?